containerfiles/initrd-builder/argfile.conf

```diff
@@ -0,0 +1,16 @@
+# podvm_payload image from where we get kata binaries artifacts, e.g. pause-bundle, kata-agent and guest components
+# Must be RHEL 10 based
```

Comment:
In addition to being "RHEL10 based", I think it would be worth explaining what it needs to be synchronized with on the RHCOS side.

Reply:
This is to pre-build initrds for baremetal coco in a container. It doesn't relate to the RHCOS version running on the node.